src/coreclr/jit/morphblock.cpp

```diff
@@ -387,6 +387,11 @@ GenTree* MorphInitBlockHelper::MorphCommaBlock(Compiler* comp, GenTreeOp* firstC
 
     GenTree* effectiveVal = lastComma->gtGetOp2();
 
+    if (effectiveVal->IsVectorConst())
```

Comment:
Should this instead be a check for `!effectiveVal->OperIs(...)` where `...` is anything that will hit `unreached` for `GT_ADDR`?

This is hitting `GT_CNS_VEC` today, but we could also hit it for a `GT_HWINTRINSIC` or another scenario, right?